notebooks/EDA.ipynb

```diff
@@ -0,0 +1,4712 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #22.            df.drop(i, axis=1, inplace=True)
</pre><p>Tutaj zamiast sprawdzać każdą kolumnę z osobna można podać od razu listę kolmumn do usunięcia. Warto zerknąć w dokumentacji, jakie argumenty przyjmuje metoda <code>drop</code>:  https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.drop.html </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DataWorkshop-Foundation/way-to-school/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>